Header header

bool collision_      #Collision happend true/flase
